<html>
<head>
<!--#include virtual="/css-scroll.txt"-->
</head>
<body>
<!--#include virtual="/nav-bar.txt"-->
</body> <head> 
  <!--#include virtual="/css-scroll.txt"-->
<style>
. {font-family:verdana,helvetica,sans-serif}
a {text-decoration:none;color:#3030a0}
</style>
</head> 
 <body>
 <div id="content">
<h2>Evaluation of Signaling Cascades Based on the Weights from Microarray and ChIP-seq Data</h2>
<p><i><b>Zerrin Isik, Volkan Atalay, Rengul Cetin-Atalay</b></i>; 
JMLR W&CP 8:44-54, 2010.</p>
<h3>Abstract</h3>

In this study, we combined the ChIP-seq and the transcriptome data and
integrated these data into signaling cascades. Integration was
realized through a framework based on data- and model-driven hybrid
approach. An enrichment model was constructed to evaluate signaling
cascades which resulted in specific cellular processes. We used
ChIP-seq and microarray data from public databases which were obtained
from HeLa cells under oxidative stress having similar experimental
setups.  Both ChIP-seq and array data were analyzed by percentile
ranking for the sake of simultaneous data integration on specific
genes. Signaling cascades from KEGG pathway database were subsequently
scored by taking sum of the individual scores of the genes involved
within the cascade. This scoring information is then transferred to en
route of the signaling cascade to form the final score. Signaling
cascade model based framework that we describe in this study is a
novel approach which calculates scores for the target process of the
analyzed signaling cascade, rather than assigning scores to gene
product nodes.


</div>
 <!--#include virtual="/nav-bar.txt"--> 
</body><p><center>Page last modified on Tue Feb  9 22:52:36 CET 2010.</center>
<body>
<p> 

<table width="100%"> <tr>
<td align="left"><font size="-1"><i><a
href="javascript:GoAddress('webmaster','jmlr.org');">webmaster<img
src=atr.gif border=0>jmlr.org</a></i></font></td> <td
align="right"><font size="-1">Copyright &copy <a target="_top"
href="http://www.jmlr.org">JMLR</a> 2000.  All rights
reserved.</font></td> </tr> </table>
</body>
