<html>
<head>
<!--#include virtual="/css-scroll.txt"-->
</head>
<body>
<!--#include virtual="/nav-bar.txt"-->
</body> <head> 
  <!--#include virtual="/css-scroll.txt"-->
<style>
. {font-family:verdana,helvetica,sans-serif}
a {text-decoration:none;color:#3030a0}
</style>
</head> 
 <body>
 <div id="content">
<h2>On utility of gene set signatures in gene expression-based cancer class prediction</h2>
<p><i><b>Minca Mramor, Marko Toplak, Gregor Leban, Toma&#382; Curk, Janez Dem&scaron;ar, Bla&#382; Zupan</b></i>; 
JMLR W&CP 8:55-64, 2010.</p>
<h3>Abstract</h3>

Machine learning methods that can use additional knowledge in their
inference process are central to the development of integrative
bioinformatics. Inclusion of background knowledge improves robustness,
predictive accuracy and interpretability. Recently, a set of such
techniques has been proposed that use information on gene sets for
supervised data mining of class-labeled microarray data sets. We here
present a new gene set-based supervised learning approach named
<i>setsig</i> and systematically investigate the predictive
accuracy of this and other gene set approaches compared to the
standard inference model where only gene expression information is
used. Our results indicate that <i>setsig</i> outperforms other
gene set approaches, but contrary to earlier reports, transformation
of gene expression data to the space of gene set signatures does not
result in increased accuracy of predictive models when compared to
those trained directly from original (not transformed) data.

</div>
 <!--#include virtual="/nav-bar.txt"--> 
</body><p><center>Page last modified on Tue Feb  9 22:52:36 CET 2010.</center>
<body>
<p> 

<table width="100%"> <tr>
<td align="left"><font size="-1"><i><a
href="javascript:GoAddress('webmaster','jmlr.org');">webmaster<img
src=atr.gif border=0>jmlr.org</a></i></font></td> <td
align="right"><font size="-1">Copyright &copy <a target="_top"
href="http://www.jmlr.org">JMLR</a> 2000.  All rights
reserved.</font></td> </tr> </table>
</body>
