---
title: On utility of gene set signatures in gene expression-based cancer class prediction
abstract: Machine learning methods that can use additional knowledge in their inference
  process are central to the development of integrative bioinformatics. Inclusion
  of background knowledge improves robustness, predictive accuracy and interpretability.
  Recently, a set of such techniques has been proposed that use information on gene
  sets for supervised data mining of class-labeled microarray data sets. We here present
  a new gene set-based supervised learning approach named \emphsetsig and systematically
  investigate the predictive accuracy of this and other gene set approaches compared
  to the standard inference model where only gene expression information is used.
  Our results indicate that \emphsetsig outperforms other gene set approaches, but
  contrary to earlier reports, transformation of gene expression data to the space
  of gene set signatures does not result in increased accuracy of predictive models
  when compared to those trained directly from original (not transformed) data.
pdf: http://proceedings.pmlr.press/mramor10a/mramor10a.pdf
layout: inproceedings
id: mramor10a
month: 0
firstpage: 55
lastpage: 64
page: 55-64
origpdf: http://jmlr.org/proceedings/papers/v8/mramor10a/mramor10a.pdf
sections: 
author:
- given: Minca
  family: Mramor
- given: Marko
  family: Toplak
- given: Gregor
  family: Leban
- given: Tomaž
  family: Curk
- given: Janez
  family: Demšar
- given: Blaž
  family: Zupan
date: 2009-03-02
address: Ljubljana, Slovenia
publisher: PMLR
container-title: Proceedings of the third International Workshop on Machine Learning
  in Systems Biology
volume: '8'
genre: inproceedings
issued:
  date-parts:
  - 2009
  - 3
  - 2
# Format based on citeproc: http://blog.martinfenner.org/2013/07/30/citeproc-yaml-for-bibliographies/
---
